tracker/views.py

```diff
@@ -0,0 +1,6 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+
+
+def hello_world(request):
+    return HttpResponse("Hello, world!")
```

Comment:
You could add a smile here :) 
```suggestion
    return HttpResponse("Hello, world! :)")
```